agents/hermes/discord-preload/sitecustomize.py

```
@@ -1,69 +1,235 @@
+
+
+_original_urllib_request_init = urllib.request.Request.__init__
+_original_urllib_add_header = urllib.request.Request.add_header
+_original_urllib_add_unredirected_header = urllib.request.Request.add_unredirected_header
+
+
+def _nemoclaw_urllib_request_init(
+    self,
+    url,
+    data=None,
+    headers={},
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)